src/server.py

```diff
@@ -131,7 +143,10 @@ def loop(self):
 
     @property
     def volume(self):
-        return self._get_multipliable_prop("volume")
+        volume = self._get_multipliable_prop("volume")
+        if volume is None:
+            volume = self._DEFAULT_VOLUME
+        return volume
```

Comment:
```python
return self._get_multipliable_prop("volume") or self._DEFAULT_VOLUME
```

Reply:
I don't think that will have quite the same effect, as 0 is an allowed volume.

Reply:
Right...

Reply:
I personally dislike the ternary conditional operator, but @ptomato comment beats taste.